geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/Runner.scala

```diff
@@ -34,9 +35,10 @@ trait Runner extends LazyLogging {
         logger.error(msg, e)
         Command.user.error(msg)
         environmentErrorInfo().foreach(Command.user.error)
-        sys.exit(-1)
-      case e: ParameterException => Command.user.error(e.getMessage); sys.exit(-1)
-      case NonFatal(e) => Command.user.error(e.getMessage, e); sys.exit(-1)
+        sys.exit(1)
+      case e: ParameterException => Command.user.error(e.getMessage); sys.exit(1)
+      case e: CommandException => Command.user.error(e.getMessage); sys.exit(1)
+      case NonFatal(e) => Command.user.error(e.getMessage, e); sys.exit(1)
```

Comment:
Hmm... any reasons to go with +1 over -1?  

I mean, I like being a positive person...

Reply:
bash exit codes have to be positive numbers. that fact bit me at some point in the past, so i tend to remember it now...

Reply:
Yeah, bash must a byte for the return code?  Looks like the values have to be between 0 and 255.